app/src/main/java/fr/free/nrw/commons/BetaConstants.kt

```diff
@@ -0,0 +1,6 @@
+package fr.free.nrw.commons
+
+object BetaConstants {
```

Comment:
If I understand correctly, this class is for constants that are used in both prod and beta?
If yes, I feel that we could find a less misleading name :-)

Reply:
@nicolas-raoul Yes, this file is for those extra constants which are needed for the beta and It will be used in both prod and beta. So either I can make sure that these constants will only be used in beta or as you said we can rename the file with a more accurate name. Which option do you think will be better?

Reply:
Could you please add detailed Javadoc explaining why this class exists, and how it is used? Thanks!